dev/io.openliberty.concurrent.internal.cdi/src/io/openliberty/concurrent/internal/cdi/ConcurrencyExtension.java

```diff
@@ -114,8 +114,14 @@ public void afterBeanDiscovery(@Observes AfterBeanDiscovery event, BeanManager b
                 event.addBean(new ManagedScheduledExecutorBean(extSvc.defaultManagedScheduledExecutorFactory, DEFAULT_QUALIFIER_SET));
 
             if (!cdi.select(ManagedThreadFactory.class, DEFAULT_QUALIFIER_ARRAY).isResolvable()) {
-                event.addBean(new ManagedThreadFactoryBean(cmd, extSvc, DEFAULT_QUALIFIER_SET));
-                producedDefaultMTF = true;
+                try {
+                    event.addBean(new ManagedThreadFactoryBean(cmd, extSvc, DEFAULT_QUALIFIER_SET));
+                    producedDefaultMTF = true;
+                } catch (IllegalStateException e) {
```

Comment:
Do we need `@FFDCIgnore(IllegalStateException.class)` for this method so that an FFDC isn't logged for this?